### ISSUE TITLE
Reques: Dynasty-scans

### ISSUE THREAD

**Issue by catthou** [briefbanane/manga-downloader#19]
I'd like to request support for dynasty scans. If this description doesn't contain enough information ask what you need and I will research it deeper.

### **Series or Anthology:**
If a work only has a single chapter, it doesn't get a description page. Multi-chapter works use the following URL schemes:
An anthology page URL: https://dynasty-scans.com/anthologies/reverie_in_a_deserted_bus
A series page URL: https://dynasty-scans.com/series/yuru_yuri

### **Chapters:**
Unfortunately, there are no reliable chapter naming scheme. Therefore, you would have to jump through hoops to download a whole work from a single chapter and I don't recommend it.

For multi-chapter works, their description page has this html tag: `<dl class="chapter-list">` where each sub `<dd>` has its own chapter URL as a link: `<a href="/chapters/kimi_no_itami_wo_kizamu_basho" class="name">`

### **Pages:**
An individual page is easy to parse once you get to it.
A page: https://dynasty-scans.com/chapters/kimi_no_itami_wo_kizamu_basho#1
The anchor at the end of the url is the page number of that chapter.

Html element of a page image: `<img src="/system/releases/000/025/423/001.jpeg" alt="001">`
The image name and alt text are not reliable, but the URL scheme is. The preceeding numbers correspond with information like chapter, category, and the first one seems to always be 000.

### **In conclusion:**
If given a /chapters URL, download just that chapter.
If given a /series or /anthologies URL, download all chapters from the chapter-list.
If given anything else, comment that the URL is not supported and recommend they open a ticket on github.
